cluster/log-dump/log-dump.sh

```diff
@@ -229,31 +229,23 @@ function save-logs-windows() {
       echo "Not saving logs for ${node}, Windows log dumping requires gcloud support"
       return
     fi
-
-    export-windows-docker-event-log "${node}"
-
-    local remote_files=()
-    for file in ${windows_node_logfiles[@]}; do
-      remote_files+=( "${WINDOWS_LOGS_DIR}\\${file}" )
-    done
-    remote_files+=( "${windows_node_otherfiles[@]}" )
-
-    # TODO(pjh, yujuhong): handle rotated logs and copying multiple files at the
-    # same time.
-    for remote_file in ${remote_files[@]}; do
-      # Retry up to 3 times to allow ssh keys to be properly propagated and
-      # stored.
-      for retry in {1..3}; do
-        if gcloud compute scp --recurse --project "${PROJECT}" \
-          --zone "${ZONE}" "${node}:${remote_file}" "${dest_dir}" \
-          > /dev/null; then
-          break
-        else
-          sleep 10
-        fi
-      done
+    # save log files from diagnostics tool.(http://go/vanadium/guest_os/guest_diagnostics_tool#pre-requirement)
+    gcloud compute instances add-metadata ${node} --metadata enable-diagnostics=true --project=${PROJECT} --zone=${ZONE}
+    local file_loc=$(gcloud alpha compute diagnose export-logs ${node} --zone=${ZONE} --project=${PROJECT} | tail -n 1)
+    local temp_local_path="${node}.zip"
+    for retry in {1..20}; do
```

Comment:
It's waiting for the diagnostics to collect all the logs and upload to GCS. And tested locally, at least 10 is enough, in case of slow network or more/bigger logs, set 20.

Reply:
Oh, so `gcloud alpha compute diagnose export-logs` is not synchronous? Maybe add a small comment to clarify this.

Reply:
yeah, it's fire and forget. Agree